.github/workflows/ci.yml

```diff
@@ -11,12 +11,6 @@ jobs:
     strategy:
       matrix:
         include:
-        - os: ubuntu-latest
```

Comment:
Sanity check: I assume the linux and mac runners need to be added back before the PR is merged?

Reply:
@HDegroote 100%, just debugging win ci